RELEASE_NOTES.md

```diff
@@ -1,5 +1,65 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Client Release Notes 14.9:
+
+## Updated Resources
+
+### group
+
+On Windows hosts the group resource now supports setting the comment field via a new `comment` property.
+
+### homebrew_cask
+
+Two issues that caused homebrew_cask to converge on each Chef run have been resolved. Thanks [@jeroenj](https://github.com/jeroenj) for this fix. Additionally the resource will no longer fail if the `cask_name` property is specified.
+
+### homebrew_tap
+
+The homebrew_tap resource no longer fails if the `tap_name` property is specified.
+
+### openssl_x509_request
+
+The openssl_x509_request resource now property writes out the CSR file if the `path` property is specified. Thank you [@cpjones](https://github.com/cpjones) for reporting this issue.
+
+### powershell_package_source
+
+powershell_package_source now suppresses warnings which prevented properly loading the resource state, and resolves idempotency issues when both the `name` and `source_name` properties were specified. Thanks [@Happycoil](https://github.com/Happycoil) for this fix.
```

Comment:
```suggestion
powershell_package_source now suppresses warnings, which prevented properly loading the resource state, and resolves idempotency issues when both the `name` and `source_name` properties were specified. Thanks [@Happycoil](https://github.com/Happycoil) for this fix.
```

Reply:
Is "idempotency" normally one word? Asking since my curiosity got the better of me.

Reply:
Yes? It's a frequently used word in config management just like that, but not anywhere else. Pretty sure Mark Burges is to blame for all of this.